app/models/preference.rb

```diff
@@ -17,6 +17,8 @@
 #  index_preferences_on_user_id  (user_id)
 #
 class Preference < ApplicationRecord
+  Preference::MAX_PREFERENCES = 5
```

Comment:
Is not necessary I think
```suggestion
MAX_PREFERENCES = 5
```